artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/Create.java

```diff
@@ -127,6 +127,10 @@
    @Option(name = "--relax-jolokia", description = "disable strict checking on jolokia-access.xml")
    private boolean relaxJolokia;
 
+
+   @Option(name = "--cloud", description = "This is an instance that will be shared by a cloud folder")
```

Comment:
this is for when an instance is intended to run as part of a docker instance.

Reply:
I did not want to call it --docker, as it could be other things such as podman.

When you use the docker image from ./artemis-docker, you may be able to pass in the artemis.home, as part of the broker.


I will convert this into its own PR.

Reply:
this is being removed.. I will rebase soon.

Reply:
Using --container comes to mind instead of --e.g docker. I actually do use podman rather than docker. I wouldn't really tie the naming here to 'cloud' or 'containers' at all though, as it seems more general than that, just a fixed option for setting instance dir locations. Not that I have a specific better suggestion. I'd maybe consider options for setting the specific locations instead though, might be more universally applicable?